test/functional/apk-utils-e2e-specs.js

```diff
@@ -69,7 +69,7 @@ describe('apk utils', function () {
   });
   describe('startApp', function () {
     it('should be able to start', async function () {
-      await adb.install(contactManagerPath);
+      await adb.install(contactManagerPath, {grantPermissions: true}, {grantPermissions: true});
```

Comment:
why twice?

Reply:
Redundant copy and replace. I'm going to clean up this code some more today I just wanted to see test results from putting in a PR